block-storage-attacher/tests/e2e/scripts/common.sh

```diff
@@ -491,3 +513,41 @@ function install_blockvolume_plugin {
 	set -e
 	echo "Checking storage plugin and driver status and wait till running"
 }
+function install_portworx_plugin {
+	if [ -z $PORTWORX_HELM_CHART ]; then
+        echo "port worx helm chart not found. Hence exiting"
+        exit 1
+    fi
+    echo "Installing helm chart portworx plugin  .."
+	# INSTALL HELM TILLER (Attempt again, if already installed)
+	echo "Initialize tiller AND Wait till running"
+	helm init --force-upgrade
+	check_pod_state "tiller-deploy"
+	
+        # INSTALL/UPGRADE HELM CHART
+	helm_values_override="--set etcdEndPoint=$ETCD_SET1;$ETCD_SET2,clusterName=$(uuidgen),etcd.credentials=$ETCDCREDS,usedrivesAndPartitions=true,usefileSystemDrive=true"
+	#helm_install_cmd="helm install $helm_values_override $PORTWROX_HELM_CHART --name portworx"
+	helm_install_cmd="helm install  $PORTWROX_HELM_CHART --name portworx"
```

Comment:
(1) The helm chart path `$PORTWROX_HELM_CHART` is having a typo. It should be set to `$PORTWORX_HELM_CHART`.

(2) The `helm_values_override` is not used. You need to use them to set values.yaml with updated values.

Reply:
done

Reply:
You said done but I see both the points given in comment are not taken care. So its not done.

Reply:
Now doing with override values